src/styles/form.scss

```diff
@@ -0,0 +1,43 @@
+.form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+p {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+form {
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+input {
+  padding: 10px;
+  margin-bottom: 1rem;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  width: 310px;
+}
+
+button {
+  padding: 10px 20px;
+  background-color: #007bff;
+  color: #fff;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+  font-size: 16px;
+
+  &:hover {
+    background-color: #0056b3;
+  }
+}
```

Comment:
It's good practice not to add styling directly to [TYPE column](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#type_column), unless it's required. It's also better better to prevent the use of that kind for CSS specificity as it will result in overriding stable code that your colleagues produce.

As you can see overriding the TYPE column styling affected existing code on the nav bar as you can see here in the photo attached.
![Screenshot 2023-10-31 232132](https://github.com/Elerqsousy/grandeur-frontend/assets/70639893/fe7d6789-4cf6-470c-a733-66ef62062586)

Kindly, change that into more specific class styling and use Scss styling tree to have more stable code.